src/interfaces/IAssetManagedPair.sol

```diff
@@ -1,8 +1,12 @@
 pragma solidity 0.8.13;
 
+import "src/interfaces/IAssetManager.sol";
```

Comment:
Can we do explicit imports, file level imports considered unnecessary

Reply:
fixed